engine/src/main/java/org/destinationsol/mercenary/MercenaryUtils.java

```diff
@@ -0,0 +1,78 @@
+package org.destinationsol.mercenary;
+
+import org.destinationsol.Const;
+import org.destinationsol.common.SolMath;
+import org.destinationsol.common.SolRandom;
+import org.destinationsol.game.Faction;
+import org.destinationsol.game.ShipConfig;
+import org.destinationsol.game.SolGame;
+import org.destinationsol.game.input.AiPilot;
+import org.destinationsol.game.input.Guardian;
+import org.destinationsol.game.item.MercItem;
+import org.destinationsol.game.planet.Planet;
+import org.destinationsol.game.ship.SolShip;
+import org.destinationsol.game.ship.hulls.HullConfig;
+
+import com.badlogic.gdx.math.Vector2;
+
+public class MercenaryUtils {
+
+    /**
+     * Creates a new mercenary, SolShip and all. It also adds the MercItem and SolShip to the
```

Comment:
Remove the `it` from the second sentence. Also, start it on the next line, and keep it all in one line.